code/weapon/weapon.h

```diff
@@ -303,6 +303,15 @@ enum class HomingAcquisitionType {
 	RANDOM,
 };
 
+struct ConditionalImpact {
+	particle::ParticleEffectHandle effect;
+	float min_health_threshold_0_to_1;
+	float max_health_threshold_0_to_1;
+	float min_angle_threshold_degrees;
+	float max_angle_threshold_degrees;
```

Comment:
Ah, to be more specific, I think it should be added as comments, you can leave the variable names as is, and indeed it is conventional to leave units out of the variable names (save for exceptional circumstances).